pool/src/storage.rs

```diff
@@ -54,6 +63,7 @@ pub struct ReserveConfig {
     pub r_three: u32, // the R3 value in the interest rate formula scaled expressed in 7 decimals
     pub reactivity: u32, // the reactivity constant for the reserve scaled expressed in 7 decimals
     pub collateral_cap: i128, // the total amount of underlying tokens that can be used as collateral
+    pub flags: u32,           // the flag of the reserve
```

Comment:
Curious if there was a reason to store this as a `u32` instead of a boolean?

At the moment we have no additional `flag` types planned, and the implementation would be simpler if we opted to do:

```
pub enabled: bool, // if the reserve is enabled. If the reserve is not enabled, positions can only be removed for this reserve.
```